sdk/src/androidTest/java/com/mapbox/maps/OfflineTest.kt

```diff
@@ -372,7 +325,7 @@ class OfflineTest {
         // verify no map loading errors occurred
         Assert.assertEquals(0, mapLoadingErrorCount)
       } else {
-        throw TimeoutException()
+        logE(TAG, "Timeout occurred, silently ignoring due to flaky nature of this test")
```

Comment:
should we add todo?
maybe track somehow?

Reply:
test removed